src/DynamoCoreWpf/Views/Preview/InfoBubbleView.xaml.cs

```diff
@@ -477,6 +492,36 @@ private TextBox GetNewTextBox(string text)
             return textBox;
         }
 
+        private TextBlock GetNewHyperlinkTextBlock(Uri uri)
+        {
+            var font = SharedDictionaryManager.DynamoModernDictionary["OpenSansRegular"];
+            TextBlock linkBlock = new TextBlock
+            {
+                TextWrapping = TextWrapping.Wrap,
+
+                Margin = ContentMargin,
+                MaxHeight = ContentMaxHeight,
+                MaxWidth = ContentMaxWidth,
+
+                Foreground = ContentForeground,
+                FontWeight = ContentFontWeight,
+                FontSize = ContentFontSize,
+
+                Background = Brushes.Transparent,
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+
+                FontFamily = font as FontFamily
+            };
+
+            Hyperlink link = new Hyperlink();
+            link.NavigateUri = uri;
+            link.RequestNavigate += RequestNavigateToDocumentationLinkHandler;
```

Comment:
can this be unsubscribed somewhere? I always eye handlers like this suspiciously - it's very difficult to tell if this will introduce a memory leak.

Reply:
done in a2b65f28740b79974ddcb5542102bb10282a7666
debugged and confirmed the viewmodel of the `InfoBubble` gets disposed when 
- the node is deleted
- the workspace is closed/cleared